packages/databricks-vscode/package.json

```diff
@@ -255,6 +255,18 @@
                 "enablement": "databricks.context.activated",
                 "category": "Databricks"
             },
+            {
+                "command": "databricks.run.dbconnect.debug",
+                "title": "Databricks Connect: Debug current file",
+                "enablement": "databricks.context.activated && databricks.context.loggedIn && databricks.context.dbconnect.enabled",
```

Comment:
Can we have these commands / menu-items available even if dbconnect is not enabled? And then make the commands prompt to enable dbconnect if they detect that it's not installed. It should save users a couple of clicks and should be less confusing than seeing a disabled menu item and knot knowing why it is disabled

Reply:
It just doesn't appear when dbconnect is disabled. But yeah makes sense. I will add that.